site/docs/v1/tech/email-templates/message-templates.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: doc
+title: Message Templates
+description: Learn how to write and manage message templates
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+FusionAuth will use message templates to send codes to users who have chosen an SMS MFA method. This is currently the only built-in workflow which uses message templates. 
+
+You may also create your own message templates with localized content and manage them using FusionAuth.
+
+* <<Managing Templates>>
+* <<Localization>>
+* <<Templates & Replacement Variables>>
+
+== Managing Templates
+
+FusionAuth ships with no templates to support the MFA workflows. You will want to create templates prior to using them in production.
```

Comment:
This is not accurate.  We are shipping with a Message Template called `Default Two Factor Request` with a default template value of `{ "defaultTemplate": "Two Factor Code: ${code}" }`.  

However, it is not assigned to SMS as the default template and that would have to be done prior to using.

Now, upon further investigation, we are NOT shipping a default *email template* for MFA.  I thought there was one but that's only in our kickstart file.  

So I'm not sure if the intent was to ship one for both email and SMS, but the current state of the codebase is only shipping an SMS message template.

Reply:
Okay. I will update and file a bug to ship a default email MFA template.

Reply:
https://github.com/FusionAuth/fusionauth-issues/issues/1195